daliuge-engine/test/manager/test_dm.py

```diff
@@ -119,7 +119,7 @@ def _test_runGraphInTwoNMs(
         drops.update(dm2._sessions[sessionId].drops)
 
         leaf_drop = drops[leaf_oid]
-        with droputils.DROPWaiterCtx(self, leaf_drop, 1):
+        with droputils.DROPWaiterCtx(self, leaf_drop, 5):
```

Comment:
I don't know why this sometimes fails, it appears to use memory drops and not plasma